### PR TITLE
GAWB-736 Passing defaultRuntimeOptions along to Cromwell

### PR DESCRIPTION
**Pull request by helgridly**
Requires a config change too.
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Make sure liquibase is updated if appropriate
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] Tell  that the PR exists if he wants to look at it **(apply requires_doge label)**
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x]  sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala [hunk 1]
```diff
@@ -73,8 +75,13 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
 
     val workflowRecs: Seq[WorkflowRecord] = setWorkflowBatchToQueued(workflowSubmission.batchSize, testData.submission1.submissionId)
 
-    assertResult(SubmitWorkflowBatch(workflowRecs.map(_.id))) {
-      Await.result(workflowSubmission.getUnlaunchedWorkflowBatch(), Duration.Inf)
+    val workflowSubMsg = Await.result(workflowSubmission.getUnlaunchedWorkflowBatch(), Duration.Inf)
```
Comment (helgridly) [inline src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala:78]: This test was failing on my machine (under H2) so I fixed it.

**Comment by helgridly** [comment]
See https://github.com/broadinstitute/firecloud-develop/pull/80 for the config change here.

**Comment by dvoet** [comment]
👍

File: src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala [hunk 2]
```diff
@@ -73,8 +75,13 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
...
+    workflowSubMsg match {
+      case SubmitWorkflowBatch(workflowIds) =>
+        assertResult( workflowRecs.map(_.id).toSet ) {
```
Comment (jmthibault79) [inline src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala]: Can use `workflowIds should contain theSameElementsAs workflowRecs.map(_.id)` instead

**Comment by jmthibault79** [comment]
👍 with or without my suggestion

File: src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala (hunk 2 above)
Comment (helgridly) [inline src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala]: Oh, that's great. Thanks!

**Pull request merged**